callHealthCheck.ts

```diff
@@ -20,12 +20,13 @@ exports.callHealthCheck = async function(){
         };
         try {
             console.log(`Calling ${FULL_URL}`)
-            const response = await axios.get(url, options);
-            console.log(response.data)
-            return response.data;
+            await axios.get(url, options);
         } catch (err) {
-            throw console.error(`Error retrieving VIP data from Fortigate: ${url}. Error : ${err} `);
+            throw console.error(`Error in CallHealthCheck Function. Error : ${err} `);
```

Comment:
console.error returns undefined, you can't throw it.

Reply:
if you are using callbacks, you need to `callback(err)` here instead of throwing anyways.

Reply:
added callback(err) and a console log for alicloud logging.